components/web/aget/Makefile

```diff
@@ -26,6 +26,8 @@ include $(WS_MAKE_RULES)/prep.mk
 include $(WS_MAKE_RULES)/justmake.mk
 include $(WS_MAKE_RULES)/ips.mk
 
+CFLAGS+= -std=c89
+
```

Comment:
It does not change the compilation with gcc 4.x since this series defaults to the c89 standard.
I can bump the revision if you want, does not change much.